src/types/config.ts

```diff
@@ -1,12 +1,12 @@
 export type Config = {
-	apiBaseUrl: string;
-	figmaJson: boolean;
-	figmaUrl: string;
-	figmaToken: string;
-	figmaTokens: boolean;
-	figmaPage: string;
-	format: "css" | "js" | "json" | "sass" | "scss";
-	outputColorFormat: "hexadecimal" | "hsl" | "hsla" | "rgb" | "rgba";
+	apiBaseUrl: string; // Base URL from Figma API
+	figmaJson: boolean; // Export or not the original JSON from Figma API
+	figmaUrl: string; // URL from the project you need to extract the tokens
+	figmaToken: string; // Your user token from Figma to connect on Figma API
+	figmaTokens: boolean; // Export or not the tokens
+	figmaPage: string; // Page from project to extract the tokens
```

Comment:
Project page to extract tokens
```suggestion
	figmaPage: string; // Project page to extract tokens
```

Reply:
4d54f77